argmin/src/core/state/iterstate.rs

```diff
@@ -930,6 +972,7 @@ where
             counts: HashMap::new(),
             time: Some(instant::Duration::new(0, 0)),
             termination_status: TerminationStatus::NotTerminated,
+            residuals: None,
```

Comment:
Another nitpick: Please move this line a couple of lines up, below `prev_jacobian`.